src/fundus/publishers/uk/the_mirror.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+from lxml.etree import XPath
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+    generic_topic_parsing,
+)
+
+class TheMirrorParser(ParserProxy):
+    class V1(BaseParser):
+        # _paragraph_selector = CSSSelector("div.lead-content__title h1")
+        _paragraph_selector = XPath("/html/body/main/article/div[2]/p[1]")
```

Comment:
```suggestion
        _paragraph_selector = XPath("/html/body/main/article/div[2]/p")
```
With the `[1]` you limit yourself to only the first p block within the container. This results to just one paragraph being extracted.